README.md

```diff
@@ -1 +1,52 @@
 # k8s-apim-operator
+
+How to try out the K8s API Manager Operator
+
+Copy the the kubectl-add.sh to /usr/local/bin/
+
+kubectl create -f deploy/namespace.yaml 
```

Comment:
SInce there are multiple directories, I think it it better to specify the location where these commands should be executed.
(navigate into apim-operator directory may be ?)